server.js

```diff
@@ -1,13 +1,11 @@
 const express = require("express");
-const cors = require("cors");
-
 const app = express();
-
-app.use(cors());
+const port = 9090;
```

Comment:
You set both `port` and `PORT` variables here - try and stick with one (and try and get it from a env var where possible as it makes it much more customisable when deploying e.g. `const PORT = process.env.PORT ?? 3000`)